src/components/BottomBar.module.scss

```diff
@@ -0,0 +1,48 @@
+@import '/src/styles/colors';
+
+.bottomBar {
+    background: #f9f9f9;
+    display: flex;
+    justify-content: end;
+    padding: 7px;
+  
+    & button {
+      font-weight: 500;
+      padding: 3px 8px;
+    }
+  
+  }
+  .addNewStationModal {
+      & p {
+          text-align: center;
+          font-size: 18px;
+          font-weight: 500;
+      }
+  }
+  .addNewStationInput {
+      height: 40px;
+      display: flex;
+      width: 336px;
+      & input, button {
```

Comment:
We try our best not to style by HTML tags! It's a bad practice. It's a small application and hence it's not that important, so I'll let it slide, but please avoid that in the future :)